docs/source/release_notes/release-notes.rst

```diff
@@ -48,28 +54,39 @@ develop
            linear in terms of stream length, as it was previously.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1341>`__)
 
+    *    - |new|
+         - AtlasDB now supports stream store compression.
+           Streams can be compressed client-side by adding the ``compressStreamInClient`` option to the stream
+           definition. Reads from the stream store will transparently decompress the data.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1357>`__)
+
+    *    - |improved|
+         - Increase connection pool idle timeout to 10 minutes, and reduce eviction check frequency to 20-30 seconds at 1/10 of connections.
+           Note that there is now a configuration called ``maxConnectionBurstSize``, which configures how large the pool is able to grow when
+           receiving a large burst of requests. Previously this was hard-coded to 5x the ``poolSize`` (which is now the default for the parameter).
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1336>`__)
```

Comment:
I had seen this too. But the 0.26.0 release explicitly says it has this PR merged. Is that something we should notify about?